schedule/yam/agama_qcow_unattended.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: agama_qcow_unattended
```

Comment:
drop technical stuff like `qcow` from name and update description please.

Reply:
Drop 'qcow' means update the existing agama_unattended.yaml to add publishing qcow for it, I just want to create a new schedule file to avoid that.

Reply:
sorry, I don't get the comment, we should only have functional names. For this case we just need to add the setting to publish qcow in existing test suite.